packages/ui/src/views/repo/pull-request/components/commit-selector/commit-selector-dropdown.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC, useMemo, useState } from 'react'
+
+import { DropdownMenuContent, DropdownMenuItem, Icon, SearchBox } from '@/components'
+import { cn } from '@utils/cn'
+
+import { CommitSelectorListItem } from '../../compare/components/types'
+import { CommitSelectorDropdownProps } from '../../types'
+
+const filterItems = (items: CommitSelectorListItem[], query: string) => {
```

Comment:
filtering should be done on the BE instead

Reply:
done

Reply:
atm since there's no filtering on be , client side is done for functionality